src/wallet/rpcwallet.cpp

```diff
@@ -1521,6 +1522,14 @@ static UniValue sendmany(const JSONRPCRequest& request)
         }
     }
 
+    if (!request.params[8].isEmpty()) {
```

Comment:
Change other optional param checks in this call to `isEmpty()` Otherwise it won't be possible to call this method with default params.